src/cart/crt_group.c

```diff
@@ -414,8 +504,9 @@ crt_grp_lc_destroy(struct crt_grp_priv *grp_priv)
 		return 0;
 
 	for (i = 0; i < CRT_SRV_CONTEXT_NUM; i++) {
-		rc2 = d_hash_table_destroy_inplace(&grp_priv->gp_lookup_cache[i],
-						   true /* force */);
+		rc2 = d_hash_table_destroy_inplace(
```

Comment:
There are several whitespace changes here, but this one appears to be a whitespace only chance that makes the indentation worse.

Reply:
this was line over 80 chars